.travis.yml

```diff
@@ -1,15 +1,30 @@
 language: node_js
 
+sudo: false
+
+os:
+  - osx
+  - linux
+
 node_js:
   - 'stable'
 
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
 install:
   - npm install
+  - npm run vscode:prepublish
 
 script:
   - tsc -p ./
+  - npm test --silent
```

Comment:
Why silent?

Reply:
That's what they had in https://code.visualstudio.com/docs/extensions/testing-extensions.  Will change.